src/components/StarterPack/Main/ProfilesList.tsx

```diff
@@ -47,7 +47,10 @@ export const ProfilesList = React.forwardRef<SectionRef, ProfilesListProps>(
     // The server returns these sorted by descending creation date, so we want to invert
     const profiles = data?.pages
       .flatMap(p => p.items.map(i => i.subject))
-      .filter(p => !p.associated?.labeler)
+      .filter(
+        p =>
+          !p.viewer?.blockedBy && !p.viewer?.blocking && !p.associated?.labeler,
```

Comment:
is this logic copypaste from somewhere? any reference points?

Reply:
idt so (in either this case or the one below). this is the one weird place where we actually don't filter the blocks out on the server because of how other lists work. we could extract it to a function if we want to though even if it doesn't get used much/ever again.

Reply:
(probably something like `isBlockedOrMuted()`?)

Reply:
did that, though made them two separate things (blocked vs muted)